Cubical/Algebra/SymmetricGroup.agda

```diff
@@ -0,0 +1,68 @@
+{-# OPTIONS --cubical --safe #-}
+module Cubical.Algebra.SymmetricGroup where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.HLevels
+open import Cubical.Data.Sigma
+open import Cubical.Data.Nat using (ℕ ; suc ; zero)
+open import Cubical.Data.Fin using (Fin ; isSetFin)
+open import Cubical.Data.Empty
+open import Cubical.Relation.Nullary using (¬_)
+
+
+
+open import Cubical.Structures.Group
+
+private
+  variable
+    ℓ ℓ' : Level
+
+
+private
+  lemma : ∀ (X : Type ℓ) → isSet X → isSet (X ≃ X)
+  lemma X X-isSet = isOfHLevelΣ 2 (isOfHLevelΠ 2 (λ _ → X-isSet)) λ f → isProp→isSet (isPropIsEquiv f)
+
+infixr 20 _∘_
+
+_∘_ : {A B C : Type ℓ} → B ≃ C → A ≃ B → A ≃ C
+g ∘ f = compEquiv f g
+
+∘-assoc : {A B C D : Type ℓ} (h : C ≃ D) (g : B ≃ C) (f : A ≃ B)
+        → h ∘ (g ∘ f) ≡ (h ∘ g) ∘ f
+∘-assoc h g f = equivEq _ _ refl
+
+-- Move computation rules of equivalences to Cubical.Foundations.Equiv?
```

Comment:
Yes, they should be moved here: https://github.com/agda/cubical/blob/master/Cubical/Foundations/Equiv.agda#L97

Reply:
Done